tests/sensors/test_time_sensor.py

```diff
@@ -63,7 +63,6 @@ def test_task_is_deferred(self):
 
         assert isinstance(exc_info.value.trigger, DateTimeTrigger)
         assert exc_info.value.trigger.moment == timezone.datetime(2020, 7, 7, 10)
-        assert exc_info.value.method_name == "execute_complete"
```

Comment:
Why do we remove this `method_name` check?

Reply:
I have removed the `execute_complete` method here in sensor.